tools/kmod/sof_remove.sh

```diff
@@ -144,7 +144,7 @@ remove_module snd_soc_acpi_intel_match
 # Machine drivers
 #-------------------------------------------
 remove_module snd_soc_sof_rt5682
-remove_module snd_soc_sof_da7219_max98373
+remove_module snd_soc_sof_da7219
```

Comment:
you want to keep the old name for backwards compatibility.
 Nothing bad will happen if you remove a nonexistent device. Bad things may happen if you keep a module loaded.

Reply:
Please add a very short comment explaining which name changed.

Reply:
to the commit message or to the script?

Reply:
commit message is fine.

Reply:
When looking at confusing code, few people think about looking at past commit messages (which are sometimes hard to find because of future whitespace changes or other formatting). Even fewer people in a repo with test scripts where commit messages tend to be lower quality.

Hopefully no one will ever have to make sense of these two lines.

Reply:
Actually, the commit title is still "kmod: sof_remove: add snd_soc_sof_da7219" even when it's not removed anymore!

https://github.com/thesofproject/sof-test/pull/1107/commits/f93eb561a03c58d5055e58b8cd51bec983085a3e

I don't mind that much if the comments are not in my preferred place. I do mind if the commit title says the opposite of what it does; please fix at least that.

Reply:
It seems to me that there is something wrong with Github's GUI.

```
brentlu@brentlu-desktop:/workspace/sof/my_sof_test_0$ git log -p f93eb56
commit f93eb561a03c58d5055e58b8cd51bec983085a3e
Author: Brent Lu <brent.lu@intel.com>
Date:   Wed Apr 10 17:26:00 2024 +0800

    kmod: sof_remove: add snd_soc_sof_da7219

    Machine driver sof_da7219_max98373 has been renamed as
    snd_soc_sof_da7219. Add the new module to the script and keep the old
    one for backward compatibility.

    Signed-off-by: Brent Lu <brent.lu@intel.com>

diff --git a/tools/kmod/sof_remove.sh b/tools/kmod/sof_remove.sh
index 4907101..b0f4fb6 100755
--- a/tools/kmod/sof_remove.sh
+++ b/tools/kmod/sof_remove.sh
@@ -152,6 +152,7 @@ remove_module imx_common
 #-------------------------------------------
 remove_module snd_soc_sof_rt5682
 remove_module snd_soc_sof_da7219_max98373
+remove_module snd_soc_sof_da7219
 remove_module snd_soc_sst_bdw_rt5677_mach
 remove_module snd_soc_bdw_rt286
 remove_module snd_soc_sst_broadwell
```